lstchain/pointing/__init__.py

```diff
@@ -0,0 +1,2 @@
+from ..pointing import *
```

Comment:
why this line if pointing has only the `pointings` module and everything is already imported?

Reply:
Should have removed this.